### PR TITLE
boards: arm: stm32g4x nucleo board at 170MHz max freq. 

### PR DESCRIPTION
**Pull request by FRASTM**
The nucleo_g474 and nucleo_g431 nucleo boards are configured with SYS_CLOCK_HW_CYCLES_PER_SEC to 170MHz. 

This requires enabling the Power Boost Mode which is already set in the `drivers/clock_control/clock_stm32g4.c` by the STM32G4x/STM32L4Rx/STM32L4Sx: Fix sys clock frequency and boost voltage for highest clock frequencies #22311

Signed-off-by: Francois Ramu francois.ramu@st.com

### CODE REVIEW

File: boards/arm/nucleo_g474re/nucleo_g474re_defconfig [hunk 1]
```diff
@@ -36,11 +35,11 @@ CONFIG_CLOCK_STM32_PLL_Q_DIVISOR=2
 CONFIG_CLOCK_STM32_PLL_R_DIVISOR=2
 
 # Produce 150MHz clock at PLLCLK output
-CONFIG_CLOCK_STM32_PLL_N_MULTIPLIER=75
+# CONFIG_CLOCK_STM32_PLL_N_MULTIPLIER=75
 
-# Comment out above and uncomment below for 170MHz. Note that you
-# must have configured the mcu for boost power mode.
-# CONFIG_CLOCK_STM32_PLL_N_MULTIPLIER=85
+# Comment out above and uncomment below for 170MHz.
```
Comment (erwango) [inline boards/arm/nucleo_g474re/nucleo_g474re_defconfig:40]: This is no longer true.

**Review by erwango — CHANGES_REQUESTED** [review]

File: boards/arm/nucleo_g474re/nucleo_g474re_defconfig (hunk 1 above)
Comment (FRASTM) [reply boards/arm/nucleo_g474re/nucleo_g474re_defconfig:40]: proposal : 

> Comment out below and uncomment above for less than 170MHz.

Comment (erwango) [reply boards/arm/nucleo_g474re/nucleo_g474re_defconfig:40]: ok, but then the note below does not apply anymore.

**Comment by FRASTM** [comment]
modified comment for setting clock below 170MHz

**Comment by FRASTM** [comment]
remove useless lines in nucleo_g4x_defconfig

**Review by erwango — APPROVED** [review]

**Review by ioannisg — APPROVED** [review]

**Pull request merged**